hera_xblocks/hera_title/hera_title/hera_title.py

```diff
@@ -13,22 +17,40 @@ class HeraTitleXBlock(StudioEditableXBlockMixin, XBlock):
     """
     editable_fields = ('data',)
 
-    display_name = String(default="Hera Title")
     data = JSONField(default={})
 
+    @property
+    def heading(self):
+        return self.data.get("heading",)
+
+    @property
+    def img_url(self):
+        return self.data.get("imgUrl", [])
```

Comment:
`imgUrl` - string with only one url to an image

Reply:
Done